muckrock/foia/models/request.py

```diff
@@ -760,8 +760,7 @@ def send_delayed_email(self, comm, **kwargs):
 
         # if we are using celery email, we want to not use it here, and use the
         # celery email backend directly.  Otherwise just use the default email backend
-        backend = getattr(settings, "CELERY_EMAIL_BACKEND", settings.EMAIL_BACKEND)
-        with get_connection(backend) as email_connection:
+        with get_connection(settings.EMAIL_BACKEND) as email_connection:
```

Comment:
Hmm are we sure we want to bring this change upstream? (I'm not sure I understand the rationale behind it.)

Reply:
the result of a long debugging session where I ultimately fixed the problem 3 different ways and this was actually no longer even necessary